### ISSUE TITLE
[Sprint] Use f-strings instead .format() function on asv/machine.py

### ISSUE THREAD

**Issue by LucyJimenez** [airspeed-velocity/asv#1176]
xref [#909](https://github.com/airspeed-velocity/asv/issues/909)

Since we support Python > 3.7 now, we can make use of f-strings, to make code more readable.

As an example, the next asv code:
`machine_env_name = "{}/{}".format(machine, env_name)`
or the next, which is equivalent:
`machine_env_name = "{0}/{1}".format(machine, env_name)`
Could be written as:
`machine_env_name = f"{machine}/{env_name}"`

(note the `f` before the string quotes)

---

The next command returns the list of strings to replace: 

 `grep -R --exclude-dir="build" --exclude="*.css" ".format(" asv/machine.py`

For this file, you will find the following issues to solve:
```
asv/machine.py:            "No information stored about machine '{0}'. I know about {1}.".format(
asv/machine.py:            'os': "{0} {1}".format(system, release),
asv/machine.py:                    '{0}. {1}: {2}'.format(
```

**Comment by asnramos** [comment]
@LucyJimenez I take this issue

**Comment by LucyJimenez** [comment]
@datapythonista, this issue has already been resolved.

**Comment by LucyJimenez** [comment]
Closed #1185
